src/libs/ck-libs/ampi/ampi.C

```diff
@@ -6092,7 +6195,7 @@ AMPI_API_IMPL(int, MPI_Wait, MPI_Request *request, MPI_Status *sts)
              *request, reqs[*request], (int)(reqs[*request]->tag));
   AMPI_DEBUG("MPI_Wait: request=%d, reqs.size=%d, &reqs=%d\n",
              *request, reqs.size(), reqs);
```

Comment:
`reqs.size()` probably won't compile (see my previous patch to restore AMPI_DEBUG)

Reply:
Fixed